tests/queries/0_stateless/02369_lost_part_intersecting_merges.sh

```diff
@@ -27,7 +27,8 @@ path=$($CLICKHOUSE_CLIENT -q "select path from system.parts where database='$CLI
 $CLICKHOUSE_CLIENT -q "select throwIf(substring('$path', 1, 1) != '/', 'Path is relative: $path')" || exit
 rm -rf $path
 
-$CLICKHOUSE_CLIENT -q "select * from rmt1;" 2>/dev/null
+$CLICKHOUSE_CLIENT -q "select * from rmt1;" 2>&1 | grep LOGICAL_ERROR
+$CLICKHOUSE_CLIENT --min_bytes_to_use_direct_io=1 --local_filesystem_read_method=pread_threadpool -q "select * from rmt1;" 2>&1 | grep LOGICAL_ERROR
```

Comment:
It reproduces only with wide parts. I added `grep LOGICAL_ERROR` to fail the test in release build because we don't have tests with wide parts and debug build.